src/org/opendatakit/briefcase/ui/SettingsPanel.java

```diff
@@ -243,15 +243,15 @@ public void actionPerformed(ActionEvent e) {
     }
 
     /**
-     * This listener will pass the user's consent to being tracked onto the
-     * application's preferences so it can be persisted and used elsewhere.
+     * This listener notifies BriefcaseAnalytics of the users updated choice
```

Comment:
I retyped it by hand. 📘 
I believe one could also go for users' in this context :).